### PR TITLE
fix: enable core command via message bus

### PR DESCRIPTION
**Pull request by marcpfuller**
Signed-off-by: Marc-Philippe Fuller <marc-philippe.fuller@intel.com>

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-rfid-llrp-inventory/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I am not introducing a breaking change (if you are, flag in conventional commit message with `BREAKING CHANGE:` describing the break)
- [x] I am not introducing a new dependency (add notes below if you are)
- [ ] I have added unit tests for the new feature or bug fix (if not, why?)
- [x] I have fully tested (add details below) this the new feature or bug fix (if not, why?)
- [ ] I have opened a PR for the related docs change (if not, why?)
  <link to docs PR>

### CODE REVIEW

**Review by jim-wang-intel — APPROVED** [review]
LGTM

File: res/configuration.toml [hunk 1]
```diff
@@ -61,9 +61,18 @@ Type = "consul"
...
   [Clients.core-command]
-  Protocol = "http"
-  Host = "localhost"
-  Port = 59882
+  #  Protocol = "http"
+  #  Host = "localhost"
+  #  Port = 59882
+  # Choose to use new messaging based communication for Command Client.
+  # Only valid if MessageBus Trigger is used
+  # If prefer REST, remove below and uncomment above
```
Comment (lenny-intel) [inline res/configuration.toml:69]: Let remove this. I left it in the template app for awareness for custom app service developers, but here we are choosing the use MessageBus.

**Review by lenny-intel — CHANGES_REQUESTED** [review]

File: res/configuration.toml (hunk 1 above)
Comment (marcpfuller) [reply res/configuration.toml]: I have removed the comments

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**